### PR TITLE
Fixes Strimzi instructions

### PR DESCRIPTION
**Pull request by deanwampler**
Version set to 0.8.2, the currently-supported version. Warning about production support made more emphatic.

### CODE REVIEW

File: src/main/paradox/includes/deploying-kafka.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
 
-[Lightbend](https://www.lightbend.com) provides a [commercially supported](https://www.lightbend.com/lightbend-platform) Strimzi installation, we recommend when going to production that you use that. The documentation for using the Lightbend supported Strimzi release can be found [here](https://developer.lightbend.com/docs/fast-data-platform/current/index.html#strimzi-operator-kafka). This guide provides a means to deploy Kafka as quickly and simply as possible for evaluation purposes, it won't give you a robust production setup.
+> WARNING: [Lightbend](https://www.lightbend.com) provides [commercial support](https://developer.lightbend.com/docs/fast-data-platform/current-OpenShift/#strimzi-operator-kafka) for Strimzi, version 0.8.2, when it is installed using the Lightbend Helm chart and instructions [discussed here](https://developer.lightbend.com/docs/fast-data-platform/current-OpenShift/#installing-strimzi). For production support, you must use this installation. The following instructions provide a means to deploy Kafka as quickly and simply as possible for evaluation purposes, but they do not provide a robust production setup.
```
Comment (TimMoore) [inline src/main/paradox/includes/deploying-kafka.md]: Paradox has special syntax for warning call outs that formats a bit more nicely: https://developer.lightbend.com/docs/paradox/current/directives/callouts.html#warning-callout

**Comment by TimMoore** [comment]
Unfortunately, the automated tests in this project are currently broken. I’ve chased it up, but have you already tested this manually?

File: src/main/paradox/includes/deploying-kafka.md (hunk 1 above)
Comment (deanwampler) [reply src/main/paradox/includes/deploying-kafka.md]: Wow. More NIH I wasn't aware of. Why not use the canonical Markdown syntax with stylesheets?

Anyway, I pushed an update with the Paradox syntax.

Comment (TimMoore) [reply src/main/paradox/includes/deploying-kafka.md]: Thanks. I wasn’t personally involved in any of the design of Paradox, but there is no standard syntax for call outs/blocks in Markdown other than inline HTML.

Comment (deanwampler) [reply src/main/paradox/includes/deploying-kafka.md]: Actually, the `> INFO: foo` is supposed to work. I don't know if it's standard Markdown, but it does work in most translators.

**Comment by TimMoore** [comment]
I ran through and tested this manually, and it worked fine, so I'll go ahead and merge despite the (unrelated) broken build.

**Pull request merged**